packages/berry-builder/sources/build-bundle.js

```diff
@@ -9,6 +9,11 @@ const dynamicLibs = require(`./dynamic-libs`);
 const findPlugins = require(`./find-plugins`);
 const makeConfig = require(`./make-config`);
 
+const pkgJsonVersion = () => {
+  const pkgJson = require("../../../package.json");
+  return `'${pkgJson["version"]}'`;
```

Comment:
```suggestion
  return JSON.stringify(pkgJson.version);
```

Reply:
I thought about this too, but thought just wrapping in quotes would be slightly faster.

Reply:
Yep I understand. I tend to favor correctness over circumstantial speed (like "faster on engine X") for stuff like this. Less surprises down the road 😄